test/brod_producer_SUITE.erl

```diff
@@ -57,7 +61,7 @@ subscriber_loop(Client, TesterPid) ->
       #kafka_message_set{ messages = Messages
                         , partition = Partition} = KMS,
       lists:foreach(fun(#kafka_message{offset = Offset, key = K, value = V}) ->
-                      TesterPid ! {Partition, K, V},
+                      TesterPid ! {Partition, Offset, K, V},
                       ok = brod:consume_ack(ConsumerPid, Offset)
                     end, Messages),
       subscriber_loop(Client, TesterPid);
```

Comment:
`Client` is unused in this function (only going to the tail call). Should I remove it?

Reply:
yes please.